### PR TITLE
fix(mexc) - unskip

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei and this too.

File: cs/ccxt/ws/Exchange.WsBridge.cs [hunk 1]
```diff
@@ -60,7 +60,25 @@ public async virtual Task loadOrderBook(WebSocketClient client, object messageHa
...
+                stored = storedRaw as ccxt.pro.OrderBook;
+            }
+            else if (storedRaw is ccxt.pro.IndexedOrderBook)
+            {
+                stored = storedRaw as ccxt.pro.IndexedOrderBook;
+            }
+            else if (storedRaw is ccxt.pro.CountedOrderBook)
+            {
+                stored = storedRaw as ccxt.pro.CountedOrderBook;
+            }
```
Comment (carlosmiei) [inline cs/ccxt/ws/Exchange.WsBridge.cs:76]: I~m not sure this cast will have any impact in practice

Comment (ttodua) [reply cs/ccxt/ws/Exchange.WsBridge.cs:76]: it does, as mentioned in previous days, casting to .OrderBook ends up with null. so, it should be casted to correct type

Comment (ttodua) [reply cs/ccxt/ws/Exchange.WsBridge.cs:76]: this update fixed the exceptions being thrown before this.

**Comment by ttodua** [comment]
@carlosmiei this should be ready, mexc is no more failing.
I'll push separate PR for skips for failing exchanges

**Pull request merged**